{"name": "test_02", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestIOzone' object has no attribute 'fp'", "trace": "self = <Case.test_fio.TestIOzone object at 0x00000219612FD1D0>\n\n    def test_02(self):\n        \"\"\"\n        \"\"\"\n>       self.fp.send('ls -l')\nE       AttributeError: 'TestIOzone' object has no attribute 'fp'\n\ntest_fio.py:14: AttributeError"}, "description": "\n        ", "start": 1740022139175, "stop": 1740022139175, "uuid": "f37d8e30-a7ae-4d7e-a156-b869a8448d8d", "historyId": "74076393987b829c40a76ca86a02226b", "testCaseId": "74076393987b829c40a76ca86a02226b", "fullName": "Case.test_fio.TestIOzone#test_02", "labels": [{"name": "parentSuite", "value": "Case"}, {"name": "suite", "value": "test_fio"}, {"name": "subSuite", "value": "TestIOzone"}, {"name": "host", "value": "Meiyi-EXT"}, {"name": "thread", "value": "24768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Case.test_fio"}]}